File: 3462-vowels-game-in-a-string/vowels-game-in-a-string.java
class Solution {
    public boolean doesAliceWin(String s) {
        int n=s.length();
        int vowelCount=0;
        for(int i=0;i<n;i++){
            char c=s.charAt(i);
            if(c=='a' || c=='e'||c=='i'||c=='o'||c=='u'){
                vowelCount+=1;
            }
        }
        if(vowelCount==0) return false;
        return true;
    }
}

File: 3462-vowels-game-in-a-string/vowels-game-in-a-string.js
var doesAliceWin = function(s) {
    let vowelCount = 0,vowel="aeiou";
    for(let i = 0 ; i < s.length ; i++){
        let c = s[i];
        if(vowel.includes(c)) vowelCount+=1;
    }

    if(vowelCount == 0) return false;
    return true;
};